code/Attack.ipynb

```diff
@@ -0,0 +1,1471 @@
+{
```

Comment:
<p>Is there a paper that discusses the training method, the architecture, etc? Additionally, which dataset was used to train the model? Please include these details. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/sayakpaul/robustness-foundation-models/pull/1/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>